### PR TITLE
Recipe step

### PR DESCRIPTION
**Pull request by OlyaMilch**

### CODE REVIEW

File: Recipe_step.py [hunk 1]
```diff
@@ -0,0 +1,63 @@
+from flask import jsonify, request
+from main import app
+from main import db
+
+
+# Определяем модель базы данных (в каком столбе какой тип данных)
+
+class Recipe_step(db.Model):
```
Comment (Makentoshe) [inline Recipe_step.py:8]: Rename as corresponding to https://peps.python.org/pep-0008/#class-names

File: Recipe_step.py [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+    id_s = db.Column(db.Integer, unique=True, primary_key=True)  # ты хочешь id в str, но точно ли нам это надо?
```
Comment (Makentoshe) [inline Recipe_step.py:9]: Let's keep it integer then until the best times

Comment (Makentoshe) [reply Recipe_step.py:9]: But then the `next step` and `prev step` also should be int
And rename 'em according to https://peps.python.org/pep-0008/#function-and-variable-names

**Review by Makentoshe — CHANGES_REQUESTED** [review]
Not bad, let's remove comments with the questions

**Review by Makentoshe — APPROVED** [review]

File: Recipe_step.py (hunk 2 above)
Comment (OlyaMilch) [reply Recipe_step.py]: fixed

File: Recipe_step.py (hunk 1 above)
Comment (OlyaMilch) [reply Recipe_step.py]: fixed

**Pull request merged**